### PR TITLE
Add stats for tomorrow

### PR DESCRIPTION
**Pull request by Strixx76**
Added all statistics attributes for tomorrows prices if they are available as per request #277

### CODE REVIEW

**Comment by renslinden** [comment]
Any update?

**Comment by utsnik** [comment]
Would love this! :)

**Review by kullarkert — APPROVED** [review]
All seems good

**Review by viiru- — APPROVED** [review]
LGTM.

File: custom_components/nordpool/sensor.py [hunk 1]
```diff
@@ -349,6 +357,24 @@ def _update(self):
         self._peak = mean(today[8:20])
         self._mean = median(today)
 
+        if self.tomorrow_valid:
+            tomorrow = self.tomorrow
+            self._tomorrow_average = mean(tomorrow)
+            self._tomorrow_min = min(tomorrow)
+            self._tomorrow_max = max(tomorrow)
+            self._tomorrow_off_peak_1 = mean(tomorrow[0:8])
```
Comment (yozik04) [inline custom_components/nordpool/sensor.py:365]: In every country off peak can be at different time.

Comment (Hellowlol) [reply custom_components/nordpool/sensor.py:365]: We use what nordpool defines as off peak but we account for local time.

**Comment by Hellowlol** [comment]
Hi,

Thanks for the PR. I will be closing this as I don't want to add any additional attributes to the sensor.

**Pull request closed without merging**